pvlib/pvarray.py

```diff
@@ -0,0 +1,219 @@
+"""
+This module contains implementations of PV module and array electrical models.
+
+These models are used to predict the electrical behavior of pv modules
+or collections of pv modules (arrays).  The primary inputs are
+effective irradiance and operating temperature and the outputs may range from
+power or efficiency at the maximum power point to complete IV curves.
+Supporting functions and parameter fitting functions may also be found here.
+"""
+
+import numpy as np
+from scipy.optimize import curve_fit
+from scipy.special import exp10
+
+
+def pvefficiency_adr(irradiance, temperature, k_a, k_d, tc_d, k_rs, k_rsh):
```

Comment:
For consistency with `calcparams_*`, should these be `effective_irradiance` and `temp_cell`?

Reply:
Same question for `fit_pvefficiency_adr`

Reply:
This comment may have been overlooked due to github's annoying habit of hiding some comments in longer reviews.  Here's the link to the comment in question: https://github.com/pvlib/pvlib-python/pull/1602#discussion_r1046336709

Reply:
I will submit to peer pressure here.